test/account.test.js

```diff
@@ -223,3 +224,32 @@ describe('with deploy contract', () => {
         expect(await contract.hello({ name: 'world' })).toEqual('hello world');
     });
 });
+
+describe('deployMultisig key rotations', () => {
+
+    test('skip access key if recovery method is "ledger"', async () => {
+        const account = await testUtils.createAccount(nearjs, 'multisig');
+        await account.addKey(KeyPair.fromRandom('ed25519').getPublicKey());
+        const originalKeys = await account.getAccessKeys();
+        const { multisig: { AccountMultisig } } = nearApi;
+        const accountMultisig = new AccountMultisig(nearjs.connection, account.accountId, {});
+        // modifiers to return the key material we want to test against
+        accountMultisig.getRecoveryMethods = () => ({
+            data: originalKeys.map(({ public_key: publicKey }, i) => ({ publicKey, kind: i > 0 ? 'ledger' : 'phrase' }))
```

Comment:
there should be non-ledger and non-phrase key as well to test that it gets converted properly